Doc for flyte agent (#971)
flyteorg/flytesnacks: cookbook/core/extend_flyte/README.rst

   * - Implement once, use in any SDK or language!
     -

=======================
External Plugin Service
=======================

> :std:ref:`External Plugin System <extend-external-plugin-system>` allows users to write a backend plugins in Python.

Reviewer: ```suggestion
:std:ref:`External Plugin System <extend-external-plugin-system>` allows you to write backend plugins in Python.
```
